src/Fmt_ast.ml

```diff
@@ -2669,9 +2669,9 @@ and fmt_signature_item c {ast= si} =
       fmt_class_types c ctx ~pre:"class type" ~sep:"=" cl
 
 and fmt_class_types c ctx ~pre ~sep (cls: class_type class_infos list) =
-  list_fl cls (fun ~first ~last cl ->
+  list_fl cls (fun ~first ~last:_ cl ->
       let {pci_virt; pci_params; pci_name; pci_expr; pci_loc; pci_attributes}
-          =
+        =
```

Comment:
Probably not intended. Happy to revert the behavior. I'm not sure I get your point regarding extra arg and indentation though.

Reply:
Re an extra arg, the code here is a bad example, but consider if you have e.g.
```
and fmt_expression c ?(box= true) ?epi ?eol ?parens ?ext ({ast= exp} as xexp)
  =
  e
```
and then add an arg `x` to the function, it will indent to
```
and fmt_expression c ?(box= true) ?epi ?eol ?parens ?ext ({ast= exp} as xexp)
    x =
  e
```
so the number of white spaces will change. Minor.

Reply:
When I was younger, I used to indent like this:
```ocaml
let some_expression foo bar baz
  onoes too_many arguments
=
  e
```

And I would still use that style if my editor wasn't fighting against me :(

Reply:
@jberdine, would you mind doing the ws change. I've limited access to my computer.

Reply:
@hhugo Done now I think.